compiler/compile/OMRSymbolReferenceTable.hpp

```diff
@@ -495,7 +495,7 @@ class SymbolReferenceTable
    TR::SymbolReference * getSymRef(int32_t i) { return baseArray.element(i); }
 
    TR::SymbolReference * createRuntimeHelper(TR_RuntimeHelper index, bool canGCandReturn, bool canGCandExcept, bool preservesAllRegisters);
-   TR::SymbolReference * findOrCreateRuntimeHelper(TR_RuntimeHelper index, bool canGCandReturn, bool canGCandExcept, bool preservesAllRegisters);
+   TR::SymbolReference * findOrCreateRuntimeHelper(TR_RuntimeHelper index, bool canGCandReturn = false, bool canGCandExcept = false, bool preservesAllRegisters = false);
```

Comment:
Thinking for future, these parameters seem almost like something that should be stored where the helper is defined. What is to stop someone from using this API passing in `false, false, false` in one place and `true, true, true` in another for the same helper. This would be incorrect then. I don't think the user should have an option when asking for the helper. We can and should make this change in the future.

Reply:
I agree these properties could be derived from a table for consistent usage.  Perhaps this idea could be documented in an issue.

Reply:
Opened up #5610.